docs/DeveloperGuide.adoc

```diff
@@ -292,20 +292,66 @@ Refer to the guide <<DevOps#, here>>.
 
 Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
 
-[width="59%",cols="22%,<23%,<25%,<30%",options="header",]
+[cols="20%,<20%,<25%,<35%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |Forgetful user |Write down the day's events | Remember what happened in that day
 
-|`* * *` |user |add a new person |
+|`* * *` |user |Have a diary  | Write my thoughts and feelings for that day
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |Unthrifty traveller |Notified if my spending goes beyond my planned levels |Adjust my budget
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |Traveller |Get my expenses to automatically converted to SGD |Avoid manual currency conversion
+
+|`* * *` |Budget traveller |Set an individual budget for each day |Minimise my spending
+
+|`* * *` |Traveller |To see amount spent each day/trip and balance left for each day/trip |Record my spending
```

Comment:
The second column is `I want to ...`, so you can remove `To` from `To see amount spent each day/trip and balance left for each day/trip`.